resources/views/profile/two-factor-authentication-form.blade.php

```diff
@@ -223,7 +223,14 @@ class="justify-center w-full"
                 </div>
                 <form class="mt-8">
                     <div class="space-y-2">
-                        <x-ark-input type="password" name="password" model="confirmedPassword" :label="trans('fortify::forms.password')" />
+                        <input type="text" autocomplete="email" hidden />
```

Comment:
@Highjhacker shouldn't this be `<input type="hidden" .... />`? assuming you meant to use a hidden input. Hidden property exists on elements but it's not how you generally make use of it for inputs

Reply:
Hmm unsure here 🤔 Think the examples I've seen about that things I've described in my comment above were using it that way, but I might be wrong. Should be `type="hidden"` instead ?

Reply:
show me an example 🤔  `type="hidden"` exists for a reason and I don't see why it wouldn't work for your scenario so 🤷